cli/src/common/boto3/ec2.py

```diff
@@ -98,6 +98,15 @@ def image_exists(self, image_name):
                 return False
             raise e
 
+    def describe_image_by_tag(self, image_name):
```

Comment:
describe_image_by_name_tag?

does it make sense to cache this call?

Reply:
Changed the function name, but I am concern about caching this call, because I am not sure we may have a case that will modify the image after first call. For example, the image was described before deletion, and if we want to get updated information of image after deletion, we can't do that if we cache the image info.